modules/account-lib/src/coin/stx/transaction.ts

```diff
@@ -47,11 +48,12 @@ export class Transaction extends BaseTransaction {
       const privKey = createStacksPrivateKey(keys.prv);
       signer.signOrigin(privKey);
     }
-    this._stxTransaction = signer.getTxInComplete();
+    //this._stxTransaction = signer.getTxInComplete();
```

Comment:
why is this commented?

Reply:
I see that TransactionSigner make changes in stxTransaction. So that code is useless.